### PR TITLE
Update README phone images

### PR DESCRIPTION
**Pull request by clivewong**
Replace outdated phone images in the README with the currently offline app experience.

### CODE REVIEW

File: images/OfflineSplash.png
Comment (dbreese) [inline images/OfflineSplash.png:1]: I think we should also change the README.md to state `Launch the Offline App`. Currently it states `Launch the mobile app`.

File: images/OfflineRecords.png
Comment (dbreese) [inline images/OfflineRecords.png:1]: I think this image should show the list of Accounts or Contacts, not the briefcase home screen.

Comment (clivewong) [reply images/OfflineRecords.png:1]: Thanks @dbreese, I'll update OfflineRecords.png to reflect your suggestion.

**Review by dbreese — CHANGES_REQUESTED** [review]
I think we need to update along with README, too as it is outdated. See comments inline.

File: images/OfflineSplash.png
Comment (clivewong) [reply images/OfflineSplash.png:1]: :+1:

File: images/OfflineDownload.png
[the remark this answers is not in the document]
Comment (clivewong) [reply images/OfflineDownload.png:1]: 👍

**Comment by ashwin-SF** [comment]
@clivewong @dbreese when previewing this branch in Git, the image layout on the README file seems a little off. They look stacked, like in the images below. Can we add line breaks between the bullet points and latest images?
Screenshot 2023-05-11 at 3 03 56 PM
Screenshot 2023-05-11 at 3 04 00 PM

File: README.md [hunk 1]
```diff
@@ -143,14 +143,21 @@ Here's an example of assigning the Edit LWC quick action for an Account:
...
+
+- Tap on **My Offline Records**:
+
+  ![Offline Download](images/LandingPage.png)
+
+- Offline Briefcase records:
+
+  ![Briefcase Records](images/MyOfflineRecords.png)
+
+- Records:
```
Comment (dbreese) [inline README.md:158]: Images look great! Wondering if we should change `Records:` to `Account Records:` or `Tap on an object (for example Accounts)`?

**Review by dbreese — APPROVED** [review]
LGTM!

**Comment by ashwin-SF** [comment]
Looks good! Let's ship it!

**Comment by ashwin-SF** [comment]
@dbreese @khawkins can we merge this PR to main?

**Review by dbreese — APPROVED** [review]
LGTM! Nice!

**Pull request merged**